Library/Homebrew/cmd/audit.rb

```diff
@@ -855,6 +855,14 @@ def audit_line(line, lineno)
       if line =~ /(require ["']formula["'])/
         problem "`#{$1}` is now unnecessary"
       end
+
+      if line =~ /#\{share\}\/#{formula.name}/
```

Comment:
Mostly just chose this way for code consistency, to be honest, similar to `if line =~ /MACOS_VERSION/` and such. It's something I place a lot of value on so it's easier for people to glance through.